browser/ui/webui/ai_chat/ai_chat_ui.cc

```diff
@@ -8,22 +8,25 @@
 #include <utility>
 
 #include "brave/browser/ui/side_panel/ai_chat/ai_chat_side_panel_utils.h"
-#include "brave/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.h"
 #include "brave/browser/ui/webui/brave_webui_source.h"
 #include "brave/components/ai_chat/browser/constants.h"
 #include "brave/components/ai_chat/common/pref_names.h"
 #include "brave/components/ai_chat/resources/page/grit/ai_chat_ui_generated_map.h"
 #include "brave/components/constants/webui_url_constants.h"
 #include "brave/components/l10n/common/localization_util.h"
 #include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/webui/webui_util.h"
 #include "components/grit/brave_components_resources.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/web_contents.h"
 #include "content/public/browser/web_ui_data_source.h"
 #include "content/public/common/url_constants.h"
 
+#if !BUILDFLAG(IS_ANDROID)
+#include "brave/browser/ui/webui/ai_chat/ai_chat_ui_page_handler.h"
```

Comment:
I'm going to work on a handler in the follow up issue as we don't have that on Android `browser_->tab_strip_model()`

Reply:
the follow up issue is https://github.com/brave/brave-browser/issues/32808